ShopAPI/ShopAPI/Controllers/ItemsController.cs

```diff
@@ -120,5 +120,26 @@ public async Task<IActionResult> DeleteProduct(int id)
 
             return NoContent();
         }
+
+        /* TODO(epadams)
+        [HttpPut("Item/UpdateStock")]
+        public async Task<IActionResult> UpdateStock(int id, int quantity)
+        {
+
+            return NoContent();
+        }
+        */
+
+        [HttpPut("Item/ChangePrice")]
+        public async Task<IActionResult> ChangePrice(int id, decimal price)
+        {
+            if (price < 0)
+            {
+                return BadRequest("Price must be 0 or greater");
+            }
+
+            await _productService.UpdatePrice(id, price);
+            return Ok("Price updated to " + price);
```

Comment:
Return the updated object, or no content.

Reply:
Done